### PR TITLE
adds assets and updates build configs

### PR DESCRIPTION
**Pull request by splendid-matt-derry**

### CODE REVIEW

File: public/assets/generic/clouds/Cloud Transparency  .docx
Comment (carlrkelly) [inline public/assets/generic/clouds/Cloud Transparency  .docx:1]: @splendid-matt-derry can we remove this?

File: src/App.css [hunk 1]
```diff
@@ -121,3 +121,97 @@ a {
...
+body {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  min-height: 568px;
+}
+
+.game {
```
Comment (carlrkelly) [inline src/App.css]: @splendid-matt-derry how do you feel like splitting this out into a game sass file apposed to in the main app.css

File: vite.config.ts [hunk 2]
```diff
@@ -10,13 +10,13 @@ export default ({ mode }) => {
...
     build: {
-      assetsDir: "assets",
+      assetsDir: "./assets",
       outDir: "dist",
       rollupOptions: {
         output: {
-          entryFileNames: `SMC-interactive-game.js`,
-          chunkFileNames: `SMC-interactive-game.js`,
-          assetFileNames: `SMC-interactive-game.[ext]`,
+          entryFileNames: `SMC-interactive-game-[hash].js`,
```
Comment (carlrkelly) [inline vite.config.ts]: We are going to have to think of a different way to hash these files as there is an external component referencing these files, and if the hash changes every build then the reference will break. maybe we can add a version query hash on our reference.

**Review by carlrkelly — CHANGES_REQUESTED** [review]

File: vite.config.ts (hunk 2 above)
Comment (splendid-matt-derry) [reply vite.config.ts]: I've updated so instead of hash we use the version specified in the package.json

File: src/Game.css [hunk 3]
```diff
@@ -0,0 +1,107 @@
+body {
```
Comment (carlrkelly) [inline src/Game.css]: This can go in the App.css as it is project wide

**Review by carlrkelly — CHANGES_REQUESTED** [review]

File: src/Game.css (hunk 3 above)
Comment (splendid-matt-derry) [reply src/Game.css]: This is resolved

**Pull request merged**